plugins/cpu/fu-cpu-common.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2025 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#include "fu-cpu-common.h"
+
+const gchar *
+fu_cpu_stream_name_to_amd_entry_sign_agesa_version(const gchar *stream_name)
```

Comment:
I think it should be clear either in function name or comment that it's the _fixed_ version stored in this lookup.
```suggestion
fu_cpu_stream_name_to_fixed_amd_entry_sign_agesa_version(const gchar *stream_name)
```